verify/verify.go

```diff
@@ -628,21 +649,25 @@ func validateX509Cert(cert *x509.Certificate, version int, signatureAlgorithm x5
 	if cert.Version != version {
 		return fmt.Errorf("certificate's version found %v. Expected %d", cert.Version, version)
 	}
+	logger.V(2).Info("Certificate version found: ", cert.Version)
```

Comment:
nit: I think just "Certificate version:" would be sufficient, WDYT? similar for below instances as well.

Reply:
Done